lib/metasploit/framework/data_service/proxy/credential_data_proxy.rb

```diff
@@ -5,7 +5,7 @@ def create_credential(opts)
       data_service = self.get_data_service()
       data_service.create_credential(opts)
     rescue Exception => e
-      puts "Call to #{data_service.class}#create_credential threw exception: #{e.message}"
+      elog "Call to #{data_service.class}#create_credential threw exception: #{e.message}"
```

Comment:
We could use `#{__callee__}` in place of the hardcoded method names. See all other similar log messages throughout the PR.